examples/docker/ci-ray-docker/build-docker-image.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+script_dir="`dirname \"$0\"`"
+pushd $script_dir
+modin_root=$(readlink -f ../../../)
```

Comment:
a comment explaining this line would be good (especially why we need `readlink`)

Reply:
`docker build` step was moved into configuration file and `readlink` call was removed, so there is no need in comment now.